### DISCUSSION TITLE
I'm a Paradox Installer if anyone has technical questions or needs unlock codes etc.

### DISCUSSION THREAD

**Discussion by elvtechnology** [ParadoxAlarmInterface/pai#411] [General]
Hey guys, figured a lot of you are going to be end users and as such won't have access to installer software and unlock codes.

I really don't want this post to come across as a sales pitch, but just saw a few people are having trouble finding support and getting unlock codes so thought I could offer some assistance.

I don't work full-time due to a disability and also don't do a lot of field work so I'm probably in a better position to assist than some other installers who focus on installations and such.

So if I can help at all, feel free to reach out. Again, really trying not to be a sales pitch but genuinely feel I can be of assistance here.

**Comment by lpaolini** [comment]
Do you have access to EVO 192 firmware changelog?
I'm wondering what's new in 7.80.3 and 7.86.1 compared to 7.70.1.

**Reply by elvtechnology** [reply]
Hey mate, I sure do. It's nothing too exciting though.

Note that I had 7.80.003 actually break PGM's that were set to 1 second on one site, so go figure. 🤷‍♂️

**7.86.001 (26 June 2025)**
Bug fix related to ACM12 communication error in rare cases.

**7.80.003 (21 June 2024)**
PGM deactivation was not operating consistently when set to 1 second timer.

**7.70.001 (1 November 2021)**
Added support for TM70 V2.00.000

**Comment by Aloe-recite** [comment]
HI! as a user, how can I see my 5050 control panel firmware version? I am looking for an IP150 to integrate it in home assistant but there aren't any availavble for sale, plus it is unknown which firmware version they come with :( is there any VERY user friendly alternative when it comes to connect Paradox to Home Assistant?

**Reply by elvtechnology** [reply]
Sorry I missed this one.

Are you referring to the IP module firmware? I imagine they would be fairly recent for new devices.

**Reply by NaanyaBiz** [reply]
i had this same problem which drove me to write the prt3 module - this is a PR at the moment, but you're welcome to take the branch and test in HA if you have a prt3 serial modeul Aloe-recite.

**Comment by timsdrew** [comment]
@elvtechnology hopefully you are able to help. Have an MG5050+  Firmware: 1.28, with an IP150 firmware: 5.6. Has been working perfectly for a couple of years, but just had installers here adding a zone. After that I cannot connect, I get the "PAI.paradox.paradox - Got ERROR Message: Partition in code lockout (too many bad entries)" error. I have tried using the user code, master code, installer code, PC code, and noting can work. Don't understand why it would change, installer insists they didn't change anything. If you have any ideas I would be grateful.

**Reply by elvtechnology** [reply]
Did they change anything while they were connected?

I do know there is a serial panel lockout feature on some panels where it will lock out connections over serial if there's too mainy failed attempts. To reset it I believe you need to arm/disarm from a KEYPAD with the system master code.

Give that a try I guess?

**Reply by timsdrew** [reply]
Thanks, will try that.

**Reply by timsdrew** [reply]
@elvtechnology have armed and disarmed with no luck. When you say master code, you mean the code we use to arm and disarm from the panel?

**Reply by timsdrew** [reply]
@elvtechnology never mind, working finally!

**Comment by pgkenya** [comment]
Any chance you can source the serial port unlock license key for my system? Let me know what information you need if so.

**Reply by elvtechnology** [reply]
I don't think I can send a DM here, but my website is my username and .com.au so you can contact me there and I'll be happy to help 🙃.

**Comment by timsdrew** [comment]
It all fixed itself after a bit. Thanks!

**Comment by niram8** [comment]
Hello, 

I am struggling to get the IP150 to work. I cannot manage to connect it to infield to install firmware v. 5.06.016 (with code IP150PCN) to use it in local net and with PAI.

I bought a P150+ mqtt with firmware 6.15.008. It is recognized by infield search, but does not connect. Apparently is stuck in opening port 10000. I use babyware v.5.6.65 with infield 5.6.55. As tested by Test-NetConnection  port 10000 has not been opened.

I can access a web interface, but stuck in the log-in. Password “paradox” or panel PC password do not work. Surprisingly at one of the numerous trials I got further connection to a web interface and could  set swan to off. But I cannot reproduce this log in with neither passwords

Overall the behaviour of the IP150+ appears not really consistent. Not every router will recognize the device and give an IP address, especially not my fritzbox 7490. 

By serial interface I can readily access the panel (MG 5050, firmware v. 4.80.14) both with babyware (5.6.65) and infield (5.6.55).

I additionally have an older IP150, firmware 4.10.000. Also cannot to infield to upgrade firmware. If this device is connected Test-NetConnection  port 10000 reports the port is open.

Apparently I lack the basic understanding how the IP150 work and how to connect them to infield for firmware update. Any idea how I can get the current IP150+ to work.

Your support is highly appreciated.

Regards

**Reply by elvtechnology** [reply]
As far as I know, the newer MQTT firmware does NOT allow inbound connections at all, even with Paradox official software. It's all outbound only through their servers or a monitoring station.

The IP150 can also be upgraded to MQTT firmware which is NOT officially reversible.

They've made a bunch of changes over the years with removing standalone and local functionality and forcing everything through their servers, some before the MQTT firmware from memory, but it's been a while so don't quote me on that.

Can you use the serial port for your integration instead? Using the IP150 just seems awfully difficult. I helped another user here and I believe that's how they got it to work.

**Reply by niram8** [reply]
Thanks for your expert help. If I understand it correctly the IP150 is not of real use for network 
access.

I want to monitor 7 paradox mg5050 on different sites remotely by babyware and PAI. I thought the IP150 would be a good choice to access by TCP over VPN. But apparently it works only with Swan, which is not an alternative.

According to your recommendation I will try to set up the remote access by serial to the MG5050. However, this will imply some programming and hardware soldering. It is not really my scope of expertise. Serial over IP could work if babyware is not very sensitive with timing.

Regards

**Reply by elvtechnology** [reply]
Yeah, Paradox really locked down the IP modules from accepting incoming connections over the years. Even if you got one on old firmware that still worked, or managed to downgrade one, it's still going to be an uphill battle.

The panel serial connector is just TTL serial and 12V power from memory. It should be easy to get something to connect to this without needing to solder anything up. Paradox even have their own serial adapter (CV4USB) for this to allow you to connect a DB9 cable to it simply, although I imagine rolling your own would be a much cheaper alternative. You might just need to do TTL to RS232 level conversion, depending on what you're connecting to at the other end.

Serial over IP actually has a good chance of working, remember these connections work over analog modems too, so they can't be too strict with timing and such. Even over IP connections in BabyWare I routinely see it stall, ask for retries etc. So it has some level of "robustness" to it, at least in BabyWare.

3rd party vendors also commonly offer serial upload/download over their own communications modules for various panels, including Paradox. This all has an IP backend also. Just remember there's various protocols and ways to configure serial over IP, and that might matter.

If you just wanted Babyware access you can also look at the IPC10/VIPC20 products as these would allow an IP module to reach them directly, and you can then reach the panels in Babyware through that connection - although I've not tried it. But also, if you're looking at the PAI route then you likely have something custom in mind that is going to fall outside the scope of this solution as well.

Hope this helps.

**Reply by NaanyaBiz** [reply]
I don't know that PRT3 would help here as it would require a serial connection to the PRT3 device itself? An alternative option?

**Reply by elvtechnology** [reply]
The PRT3 route is also an option, but I don't think it would be bring much benefit over native panel serial?

**Reply by NaanyaBiz** [reply]
If you can get native serial to work that's perfect. The PRT3 interface is well documented and supported, not encrypted so for now, it is an integration of choice for other platforms (control4 for example). I had one for a control4 hybrid but found it super clunky so ended up writing the PRT3 interface. But def try native because that's a lesser cost straight away.

**Comment by eleazerarroliga32-stack** [comment]
Master buenas tardes

Tengo una duda, estoy intentando integrar sitios a una central de alarma IPC10, pero el problema es que se actualizan los IP150+ a la version 6.15.008 MQTT, dentro de la misma red conectado el ip y el ipc10 este se registra, pero luego dentro de la misma red solo que en otro cisco este no se logra registrar aunque esten dentro de la misma red, es posible que este equipo no pueda manejar la conexionen en L3 red enrutada/segmentada ya que agote todas la pruebas
